components/base/kinematicbase/kinematics.go

```diff
@@ -23,6 +23,8 @@ type KinematicBase interface {
 	referenceframe.InputEnabled
 
 	Kinematics() referenceframe.Frame
+	ExecutionFrame() referenceframe.Frame
```

Comment:
IMO there should be a better name for this

Reply:
I will think of a better name. Maybe `ExecutionKinematics`?
@raybjork lmk if you have any opinions on this.

Reply:
Kinematics is not really correct here the frame has nothing to do with kinematics of the base

Maybe something related to its localization, since thats the transformation the frame is responsible for making?  Perhaps `LocalizationFrame` or something similar?  Open to other suggestions.